.github/workflows/release.yml

```diff
@@ -131,9 +137,10 @@ jobs:
         env:
           RELEASE_NOTES: ${{ fromJSON(steps.get-release.outputs.result).body }}
           RELEASE_URL: ${{ fromJSON(steps.get-release.outputs.result).url }}
+          COMMIT_DETAILS: ${{ needs.version-tag.outputs.commit_details }}
         run: |
           # Add additional line before the content
-          RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}"
+          RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${COMMIT_DETAILS}"
   
```

Comment:
When setting the output for commit_details, make sure to use the correct format for the output variable. The correct format is `echo "::set-output name=commit_details::$COMMIT_DETAILS"`.